client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -209,9 +209,9 @@ class DomainManagementNavigationEnhanced extends React.Component {
 		if ( ! privacyAvailable ) {
 			description = translate( 'Privacy protection: not available' );
 		} else if ( privateDomain ) {
-			description = translate( 'Privacy protection: yes' );
+			description = translate( 'Privacy protection: on' );
 		} else {
-			description = translate( 'Privacy protection: no' );
+			description = translate( 'Privacy protection: off' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Privacy Protection' )` *ES Score: 6*